draft-ietf-scitt-architecture.md

```diff
@@ -482,25 +482,23 @@ Might dereference to:
 }
 ~~~
 
-### Naming Artifacts
+### Support for Multiple Artifacts
 
-Many Issuers issue Signed Statements about different Artifacts under the same DID, so it is important for everyone to be able to immediately recognize by looking at the Envelope of a Signed Statements what Artifact it is referring to.
+Many Issuers produce Signed Statements about various Artifacts under the same Identity. Issuers and Verifiers must be able to recognize the Artifact to which the statements pertain by looking at the Signed Statement.
 This information is stored in the Feed header of the Envelope.
 Issuers MAY use different signing keys (identified by `kid` in the resolved key manifest) for different Artifacts, or sign all Signed Statements under the same key.
 
-### Signed Statement Metadata
+### Registration Policy Metadata
 
-Besides Issuer, Feed and kid, the only other mandatory metadata in a Signed Statement is the type of the Payload, indicated in the `cty` (content type) Envelope header.
-However, this set of mandatory metadata is not sufficient to express many important Registration Policies.
-For example, a Registry may only allow a Signed Statement to be registered, if it was signed recently.
-While the Issuer is free to add any information in the payload of the Signed Statements, the Transparency Services (and most of its Auditors) can only be expected to interpret information in the Envelope.
+SCITT payloads are opaque to Transparency Services, so Registration Policy decisions can only be based on interpretation of information in the Envelope.
 
-Such metadata, meant to be interpreted by the Transparency Services during Registration Policy evaluation, SHOULD be added to the `reg_info` header, unless the data is private (in which case, it MAY be sent to the Transparency Service as an additional input during registration).
+The small mandatory set of metadata in the envelope of a Signed Statement is neither intended nor sufficient to express the information required for the processing of Registration Policies in a Transparency Service.
 
-While the header MUST be present in all Signed Statements, its contents consist of a map of named attributes.
-Some attributes (such as the Issuer's timestamp) are standardized with a defined type, to help uniformize their semantics across Transparency Services.
-Others are completely customizable and may have arbitrary types.
-In any case, all attributes are optional; so the map MAY be empty.
+For example, a Registry may only allow a Signed Statement to be registered if it was signed recently very recently, or may reject a Signed Statement if it arrives out of order in some sequenced protocol.
```

Comment:
Why "recently very recently"?

Reply:
Is this a shorthand way of alluding to the epoch works from the RATS WG? If so, should we word appropriately and reference that as one potential temporal approach, which can be substituted for others? (I am guessing here based on wording, but I could be very wrong).

https://datatracker.ietf.org/doc/draft-birkholz-rats-epoch-markers/05/

Reply:
```suggestion
For example, a Registry may only allow a Signed Statement to be registered if it was signed very recently, or may reject a Signed Statement if it arrives out of order in some sequenced protocol.
```
Correcting, what may, or may not have been an intentional duplication of "_recently_"